build/tools/FileSystem.cs

```diff
@@ -8,12 +8,6 @@ public static class FileSystem
     {
         public static string[] GetFiles(AbsolutePath path, string filter)
         {
-            if (!Directory.Exists(path))
-            {
-                Directory.CreateDirectory(path!);
-                return Array.Empty<string>();
-            }
-
```

Comment:
At this point, does the `FileSystem` class need to exist? Seems like unnecessary obfuscation now? Is there an argument for keeping it around?